src/media/js/app.js

```diff
@@ -51,6 +52,7 @@ const reducer = combineReducers({
   addonReview,
   addonSubmit,
   addonSubmitVersion,
+  addonThread,
```

Comment:
i think it's important to have these things all in the main file to visualize the app without having to jump around files. adding a reducer is often accompanied by adding a route/container so we'd like to keep these changes in one area